javascript/modules/client/README.md

```diff
@@ -1921,6 +2016,26 @@ console.log(params)
 }
 */
 ```
+### Decode Mint Token Action (ERC-20)
```

Comment:
Why are the 3 new encoding actions together but not the 3 new decoding ones? Adding things in a predictable place makes them easier to find.